profiles/util.py

```diff
@@ -59,3 +66,17 @@ def format_gravatar_url(user_email, size):
             parse.quote_plus(default_image_url)
         )
     )
+
+
+def profile_image_upload_uri(_, filename):
+    """
+    Helper to format the uri for the profile image upload
+    """
+    name, ext = path.splitext(filename)
+    timestamp = datetime.now(pytz.utc).replace(microsecond=0)
+
+    return "profile/{name}-{timestamp}{ext}".format(
+        name=name[:IMAGE_FILENAME_MAX_LENGTH],
+        timestamp=timestamp.strftime("%Y-%m-%dT%H%M%S%z"),
```

Comment:
are you just going for iso format here? pretty sure you can use `timestamp.isoformat()`. if that's not what you're going for, you can leave as is